frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -72,51 +110,42 @@ export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = (
     });
   };
 
+  const handleResourceSelect = (selectedName: string) => {
+    setConfigSecretName('');
+    const newKeys = getKeysByName(existingCertConfigMaps, existingCertSecrets, selectedName);
+    setExistingCertKeys(newKeys); // Update keys for the second dropdown for existing certificate field
+
+    const newInfo = {
+      ...secureDBInfo,
+      configMap: selectedName,
+      key: '',
+    };
+
+    setSecureDBInfo({ ...newInfo, isValid: isValid(newInfo) });
```

Comment:
I think we need to modify `SecureDBInfo` to include whether the selected resource is a ConfigMap or a Secret. That'll be needed for the submit logic.

Reply:
Dang, also we probably shouldn't have `isValid` in state but that's probably out of the scope of this PR. I missed that when reviewing the last PR.